doc/tutorials/enable_s5.rst

```diff
@@ -70,17 +70,17 @@ Trigger the User VM's S5
 
 On the Service VM side, it uses the ``acrnctl`` tool to trigger the User VM's S5 flow:
 ``acrnctl stop user-vm-name``. Then, the Device Model sends a ``shutdown`` command
-to the User VM through a channel. If the User VM receives the command, it will send an "ACK"
+to the User VM through a channel. If the User VM receives the command, it will send an ``ACKED``
 to the Device Model. It is the Service VM's responsibility to check if the User VMs
-shut down successfully or not, and decides when to power off itself.
+shut down successfully or not, and decides when to power itself off.
 
 User VM "Lifecycle Manager"
 ===========================
 
-As part of the current S5 reference design, a lifecycle manager daemon (life_mngr) runs in the
-User VM to implement S5. It waits for the command from the Service VM on the
-paired serial port. The simple protocol between the Service VM and User VM is as follows:
-When the daemon receives ``shutdown``, it sends "ACKed" to the Service VM;
+As part of the current S5 reference design, a Lifecycle Manager daemon (``life_mngr`` in Linux,
```

Comment:
I'd delete "current" since all the documentation is talking about the current design and we generally avoid talking about what we may do in the future (no promises).